src/GasEstimation/BlockScaleFetchingService.ts

```diff
@@ -14,11 +14,16 @@ export class BlockScaleFetchingService implements IGasPriceFetchingService {
 
     const json = await response.json();
 
+    const toGwei = (val: number) => {
+      const gwei = 1000000000;
+      return val * gwei;
+    };
```

Comment:
Can't we use 'web3.fromWei(1000000000, 'gwei')`?

Reply:
Yeah this function is actually named wrong. It should be `web3.toWei(val, 'gwei')`

Reply:
True yeah, sry.